src/decode.rs

```diff
@@ -12934,7 +12864,7 @@ unsafe fn decode_b(
                         t,
                         lowest_px,
                         0 as libc::c_int,
-                        b_dim,
+                        b_dim.as_ptr(),
```

Comment:
Didn't I change `obmc_lowest_px` to take `b_dim` as `&[u8; 4]` and not a pointer already?  Or is this PR just branched off before I made that change?

Reply:
Yup, that's fixed up after a rebase.